invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/accordion.overrides

```diff
@@ -3,6 +3,15 @@
   &.tab-menu-accordion {
     transform: none;
   }
+
+  transform: rotate(90deg);
```

Comment:
Maybe we can use directly the preset [variable](https://github.com/Semantic-Org/Semantic-UI/blob/597843ab84d565cccfd8fd7719416350ae73e734/src/themes/default/modules/accordion.variables#L51) from Semantic UI?